app/services/work_order_delivery_service.rb

```diff
@@ -0,0 +1,61 @@
+class WorkOrderDeliveryService
+  def initialize(work_order, current_user)
+    @work_order = work_order
+    @current_user = current_user
+  end
+
+  def deliver!
```

Comment:
On a last meeting i've heard that Connor and Brad had issues with emails before. Maybe we should add some smoke test here, let's say that after running this service we sent out 6 messages? It would be nice if we will actually try to send them (so we can verify that email templates have been rendered properly as well), usually people prefer to only check mail queue as verification.

Reply:
Could you explain a bit more what you're proposing?